app/libs/commonCrudOperations.js

```diff
@@ -1,4 +1,5 @@
 import pick from '../libs/pick';
+import { useId } from '../libs/useId';
```

Comment:
useId naming is meaningless,   const select = useId(req, findBy);  (please look into the usage of this function)

Also, it would be great to avoid creating independent modules/ functions for such tiny logic, please declare this function right in commonCrudOperations.js